src/locale/dim.json

```diff
@@ -20,13 +20,13 @@
     "Beta": "BETA",
     "ButtonOff": "Inventory Mode",
     "ButtonOn": "Active Mode",
+    "ChangeFilter": "Change Filter",
     "Farming": "Recent Items",
     "FarmingStart": "Start Farming",
     "Loadouts": "Loadouts",
     "NoQuests": "You're not tracking any quests",
-    "Pursuits": "Pursuits",
-    "ChangeFilter": "Change Filter",
```

Comment:
Odd that this one wasn't reordered previously.

Reply:
yeah, this happened when I tried running `yarn lint` to fix my problems with prettier not working in code